diff --git a/java/org/l2jmobius/gameserver/Shutdown.java b/java/org/l2jmobius/gameserver/Shutdown.java
index 33db2ae6c..1f778d52d 100644
--- a/java/org/l2jmobius/gameserver/Shutdown.java
+++ b/java/org/l2jmobius/gameserver/Shutdown.java
@@ -262,6 +263,15 @@ public class Shutdown extends Thread
 		_shutdownMode = mode;
 	}
 	
+	/**
+	 * Get shutdown mode.
+	 * @return the shutdown mode
+	 */
+	public int getMode()
+	{
+		return _shutdownMode;
+	}
+	
 	/**
 	 * Set shutdown mode to ABORT.
 	 */

