diff --git a/java/org/l2jmobius/gameserver/Shutdown.java b/java/org/l2jmobius/gameserver/Shutdown.java
index 55f2710f..6232f25a 100644
--- a/java/org/l2jmobius/gameserver/Shutdown.java
+++ b/java/org/l2jmobius/gameserver/Shutdown.java
@@ -266,6 +266,15 @@ private void setMode(int mode)
 		_shutdownMode = mode;
 	}
 	
+	/**
+	 * Get shutdown mode.
+	 * @return the shutdown mode
+	 */
+	public int getMode()
+	{
+		return _shutdownMode;
+	}
+	
 	/**
 	 * Set shutdown mode to ABORT.
 	 */