--- LoginServerThread.java
+++ LoginServerThread.java
@@ -57,6 +57,7 @@
 import org.l2jmobius.gameserver.network.loginserverpackets.game.ChangeAccessLevel;
 import org.l2jmobius.gameserver.network.loginserverpackets.game.ChangePassword;
+import org.l2jmobius.gameserver.network.loginserverpackets.game.FiskPayResponseSend;
 import org.l2jmobius.gameserver.network.loginserverpackets.game.PlayerAuthRequest;
 import org.l2jmobius.gameserver.network.loginserverpackets.game.PlayerInGame;
 import org.l2jmobius.gameserver.network.loginserverpackets.game.PlayerLogout;
@@ -66,6 +67,7 @@
 import org.l2jmobius.gameserver.network.loginserverpackets.game.TempBan;
 import org.l2jmobius.gameserver.network.loginserverpackets.login.AuthResponse;
 import org.l2jmobius.gameserver.network.loginserverpackets.login.ChangePasswordResponse;
+import org.l2jmobius.gameserver.network.loginserverpackets.login.FiskPayRequestReceive;
 import org.l2jmobius.gameserver.network.loginserverpackets.login.InitLS;
 import org.l2jmobius.gameserver.network.loginserverpackets.login.KickPlayer;
 import org.l2jmobius.gameserver.network.loginserverpackets.login.LoginServerFail;
@@ -329,6 +331,11 @@
						case 0x06:
						{
 							new ChangePasswordResponse(incoming);
 							break;
 						}
+						case 0x07:
+						{
+							new FiskPayRequestReceive(incoming);
+							break;
+						}
 					}
@@ -660,6 +667,11 @@
	public void sendChangePassword(String accountName, String charName, String oldpass, String newpass)
 	{
 		sendPacket(new ChangePassword(accountName, charName, oldpass, newpass));
 	}
+
+	public void sendFiskPayResponse(int requestId, String response)
+	{
+		sendPacket(new FiskPayResponseSend(requestId, response));
+	}
 	
 	public int getServerStatus()
 	{