--- Config.java
+++ Config.java
@@ -93,5 +93,7 @@
	public static final String SIEGE_CONFIG_FILE = "./config/Siege.ini";
	public static final String TW_CONFIG_FILE = "./config/TerritoryWar.ini";
	
+	private static final String BLOCKCHAIN_CONFIG_FILE = "./config/Blockchain.ini";
+
	private static final String CHARACTER_CONFIG_FILE = "./config/Character.ini";
	private static final String CH_SIEGE_CONFIG_FILE = "./config/ConquerableHallSiege.ini";
@@ -977,6 +979,12 @@
	// --------------------------------------------------
	// No classification assigned to the following yet
	// --------------------------------------------------
	public static int MAX_ITEM_IN_PACKET;
+
+	public static boolean BLOCKCHAIN_ENABLE;
+	public static String BLOCKCHAIN_SYMBOL;
+	public static String BLOCKCHAIN_WALLET;
+	public static String BLOCKCHAIN_PASSWORD;
+
	public static int GAME_SERVER_LOGIN_PORT;
	public static String GAME_SERVER_LOGIN_HOST;
@@ -3330,4 +3338,10 @@
		else if (SERVER_MODE == ServerMode.LOGIN)
		{
+			final PropertiesParser blockchainConfig = new PropertiesParser(BLOCKCHAIN_CONFIG_FILE);
+			BLOCKCHAIN_ENABLE = blockchainConfig.getBoolean("Enable", false);
+			BLOCKCHAIN_SYMBOL = blockchainConfig.getString("Symbol", "USD");
+			BLOCKCHAIN_WALLET = blockchainConfig.getString("Wallet", "0x0000000000000000000000000000000000000000");
+			BLOCKCHAIN_PASSWORD = blockchainConfig.getString("Password", ""); 
+
			final PropertiesParser loginConfig = new PropertiesParser(LOGIN_CONFIG_FILE);
			GAME_SERVER_LOGIN_HOST = loginConfig.getString("LoginHostname", "127.0.0.1");