--- GameServerListener.java
+++ GameServerListener.java
@@ -9,6 +9,8 @@
     public void addClient(Socket s)
     {
+        GameServerThread gst = new GameServerThread(s);
         _gameServers.add(gst);
+        FiskPayLoginClient.getInstance().updateServers(gst.getServerId());
     }
 
     public void removeGameServer(GameServerThread gst)
     {
         _gameServers.remove(gst);
+        FiskPayLoginClient.getInstance().updateServers(gst.getServerId());
     }
